workers/handshake-manager/src/manager/price_agreement.rs

```diff
@@ -86,7 +33,10 @@ lazy_static! {
 pub fn init_price_streams(
     price_reporter_job_queue: PriceReporterQueue,
 ) -> Result<(), HandshakeManagerError> {
-    for (base, quote) in DEFAULT_PAIRS.iter() {
+    let quote = Token::from_ticker(USDC_TICKER);
+
+    for (addr, _) in TOKEN_REMAPS.get().unwrap().iter() {
```

Comment:
we should probably filter out USDC from this list, here and below (not sure the behavior when fetching USDC-USDC price)

Reply:
good point